spec/classes/datadog_agent_integrations_http_check_spec.rb

```diff
@@ -90,6 +92,7 @@
         it { should contain_file(conf_file).with_content(%r{threshold: 456}) }
         it { should contain_file(conf_file).with_content(%r{window: 789}) }
         it { should contain_file(conf_file).with_content(%r{content_match: 'foomatch'}) }
+        it { should contain_file(conf_file).with_content(%r{reverse_content_match: false}) }
```

Comment:
should be: `reverse_content_match: true`

Reply:
Fixed